apps/chat/src/utils/server/chat.ts

```diff
@@ -119,33 +129,29 @@ export const chatErrorHandler = ({
 }: {
   error: DialAIError | unknown;
   res: NextApiResponse;
-  msg?: string;
+  msg: string;
   isStreamingError?: boolean;
 }): void => {
   const postfix = isStreamingError ? '\0' : '';
   const fieldName = isStreamingError ? 'errorMessage' : 'message';
+  let fallbackErrorMessage = errorsMessages.generalServer;
 
   logger.error(error, msg);
+
   if (error instanceof DialAIError) {
     // Rate limit errors and gateway errors https://platform.openai.com/docs/guides/error-codes/api-errors
     if (['429', '504'].includes(error.code)) {
-      return res
-        .status(500)
-        .send(JSON.stringify({ [fieldName]: errorsMessages[429] }) + postfix);
-    }
-    if (error.code === 'content_filter') {
-      return res
-        .status(500)
-        .send(
-          JSON.stringify({ [fieldName]: errorsMessages.contentFiltering }) +
-            postfix,
-        );
+      fallbackErrorMessage = errorsMessages[429];
```

Comment:
Maybe should be by code?
```suggestion
      fallbackErrorMessage = errorsMessages[error.code];
```

Reply:
We have same error for 429 and 504 error, not sure that I need to duplicate it in constants